dbcs/honda_crv_2017_can.dbc

```diff
@@ -0,0 +1,301 @@
+VERSION ""
+
+
+NS_ :
+	NS_DESC_
+	CM_
+	BA_DEF_
+	BA_
+	VAL_
+	CAT_DEF_
+	CAT_
+	FILTER
+	BA_DEF_DEF_
+	EV_DATA_
+	ENVVAR_DATA_
+	SGTYPE_
+	SGTYPE_VAL_
+	BA_DEF_SGTYPE_
+	BA_SGTYPE_
+	SIG_TYPE_REF_
+	VAL_TABLE_
+	SIG_GROUP_
+	SIG_VALTYPE_
+	SIGTYPE_VALTYPE_
+	BO_TX_BU_
+	BA_DEF_REL_
+	BA_REL_
+	BA_DEF_DEF_REL_
+	BU_SG_REL_
+	BU_EV_REL_
+	BU_BO_REL_
+	SG_MUL_VAL_
+
+BS_:
+
+BU_: EBCM NEO ADAS PCM EPS VSA SCM BDY XXX EPB
+
+BO_ 57 XXX_1: 3 XXX
+
+BO_ 148 XXX_2: 8 XXX
+ SG_ LAT_ACCEL : 7|10@0+ (0.02,-512) [-20|20] "m/s2"  NEO
+ SG_ LONG_ACCEL : 24|9@0- (-0.02,0) [-20|20] "m/s2"  NEO
+
+BO_ 199 XXX_3: 4 XXX
+ SG_ BRAKE_PRESSURE1 : 7|10@0+ (0.015625,-103) [0|1000] ""  NEO
+ SG_ BRAKE_PRESSURE2 : 9|10@0+ (0.015625,-103) [0|1000] ""  NEO
+
+BO_ 231 XXX_4: 5 ADAS
```

Comment:
Was expecting 231(0xE7) to be STEERING_CONTROL since message ID's were close with same message length, but can't be, this message seems to change when braking.